### ISSUE TITLE
Standard Go project structure

### ISSUE THREAD

**Issue by erikgb** [rubensseva/kafgo#1]
https://github.com/golang-standards/project-layout#readme

**Comment by rubensseva** [comment]
The project layout has been changed to more closely resemble the standard. 

Still a bit unsure how to handle the protofiles. I think a makefile in the root of the repo that puts the generated Golang code under `pkg/` would be appropriate. 

The changes where included in this PR: https://github.com/rubensseva/kafgo/pull/3
The PR is mostly about testing, but it also changes the file structure.

**Comment by erikgb** [comment]
I think I would put the proto-files in the `/api` directory, ref. https://github.com/golang-standards/project-layout/tree/master/api. And I also think it could make sense to keep the generated Golang code somewhere in the same structure.

**Comment by rubensseva** [comment]
Makes sense!
Just to be clear, this means that the generated Go code should also be under the `/api` directory?

**Comment by erikgb** [comment]
> Makes sense! Just to be clear, this means that the generated Go code should also be under the `/api` directory?

Yes, I think that could make sense. Probably in a sub-directory/package.
